Source/SwiftLintFramework/Extensions/NSRegularExpression+SwiftLint.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  NSRegularExpression+SwiftLint.swift
+//  SwiftLint
+//
+//  Created by Scott Hoyt on 1/21/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public extension NSRegularExpression {
+    public convenience init(pattern: String) throws {
+        try self.init(pattern: pattern,
+            options: [.AnchorsMatchLines, .DotMatchesLineSeparators])
+    }
+    public static func forcePattern(pattern: String) -> NSRegularExpression {
```

Comment:
There's also a comment on `regex()` stating something like "safe to force cast because all regexes used in SwiftLint are verified to work" or something which would make sense to restate here IMO.